src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -112,12 +111,10 @@ export class TechRecordSummaryComponent implements OnInit {
       /*  7 */ PsvBrakeSectionWheelsHalfLocked,
       /*  8 */ PsvDdaTemplate,
       /*  9 */ DocumentsTemplate,
-      /* 10 */ getBodySection(VehicleTypes.PSV),
-      /* 11 */ getGrossVehicleWeightsSection(VehicleTypes.PSV),
-      /* 12 */ PsvTrainWeight,
-      /* 13 */ getAxleWeightsSection(VehicleTypes.PSV),
-      /* 14 */ getTyresSection(VehicleTypes.PSV),
-      /* 15 */ getDimensionsSection(VehicleTypes.PSV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord.dimensions?.axleSpacing),
+      /* 10 */ PsvWeight,
+      /* 11 */ getBodySection(VehicleTypes.PSV),
+      /* 12 */ getTyresSection(VehicleTypes.PSV),
+      /* 13 */ getDimensionsSection(VehicleTypes.PSV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord.dimensions?.axleSpacing)
```

Comment:
Have things broken here with resolving merge conflicts?

Reply:
What is broken sorry? This seems to work fine for me?